### DISCUSSION TITLE
[this is another question] How can I start the extension after the solution is loaded?

### DISCUSSION THREAD

**Discussion by Smartisa** [davidjohnoliver/CodeConnections#11] [Q&A]
When starting the solution, the extension runs automatically, and errors often occur. How can I achieve the same effect as you?

**Comment by davidjohnoliver** [comment]
What sort of errors do you observe?

In general, the extension will load when its tool window is opened, and will stay in memory until Visual Studio is closed. If you wanted it to be loaded after the solution, you would need to close the extension window, restart VS, open the solution, and then show the extension window.

But the order should not matter; if it does, it sounds like there's a bug.
